### PR TITLE
feat: add warp route monitoring for new Eclipse deploys

### PR DESCRIPTION
**Pull request by tkporter**
### Description

- Updates the monitoring script to support native balance getting and synthetic total supply getting
- Drive-by to update `tokenProgramPubKey` -> `tokenMintPubKey`, which is more accurate (the mint isn't a program)

### CODE REVIEW

**Review by paulbalaji — APPROVED** [review]
changes lgtm but would appreciate @Mo-Hussain having a gander too

File: typescript/infra/config/environments/mainnet3/warp/eclipse-SOL-deployments.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Configs and artifacts for the deployment of Hyperlane Warp Routes
```
Comment (Mo-Hussain) [inline typescript/infra/config/environments/mainnet3/warp/eclipse-SOL-deployments.yaml:1]: Did you craft these manually?

Comment (Mo-Hussain) [reply typescript/infra/config/environments/mainnet3/warp/eclipse-SOL-deployments.yaml:1]: Is there a way I can double check these configs? where did it come from?

**Review by Mo-Hussain — APPROVED** [review]

File: typescript/infra/config/environments/mainnet3/warp/eclipse-SOL-deployments.yaml (hunk 1 above)
Comment (tkporter) [reply typescript/infra/config/environments/mainnet3/warp/eclipse-SOL-deployments.yaml:1]: yeah I did, I crafted them using values from the registry. I've also ran with the configs so feel good they're accurate

**Pull request merged**